File: Aziz-Angular/src/app/models/feemanagementrequestdto.ts
export class FeeManagementRequestDTO {
    fee_name?:any;
    fee_amount?:any;
    classNames?:any[];
    start_date?:any;
    end_date?:any;
    monthName?:any;
    className?:any;
    paymentStatus?:any;
    studentId?:any;
    paymentMethod?:any;
    paymentDate?:any;

    constructor(fee_name:any, fee_amount:any, classNames:any, start_date:any, end_date:any, monthName:any,className?:any,
            paymentStatus?:any, studentId?:any, paymentMethod?:any, paymentDate?:any) {
        this.fee_name = fee_name;
        this.fee_amount = fee_amount;
        this.classNames = classNames;
        this.start_date = start_date;
        this.end_date = end_date;
        this.monthName = monthName;
        this.className = className;
        this.paymentStatus = paymentStatus;
        this.studentId = studentId;
        this.paymentMethod = paymentMethod;
        this.paymentDate = paymentDate;
        
    }
}

File: Aziz-Spring/src/main/java/com/example/demo/FeeManagementRequestDTO.java
package com.example.demo;

import java.util.List;

public class FeeManagementRequestDTO {
	String fee_name;
	int fee_amount;
	List<String> classNames;
	String start_date;
	String end_date;
	String monthName;
    String className;
    String paymentStatus;
    int studentId;
    String paymentMethod;
    String paymentDate;
	
	public FeeManagementRequestDTO() {
		super();
	}

	public String getFee_name() {
		return fee_name;
	}

	public void setFee_name(String fee_name) {
		this.fee_name = fee_name;
	}

	public int getFee_amount() {
		return fee_amount;
	}

	public void setFee_amount(int fee_amount) {
		this.fee_amount = fee_amount;
	}

	public List<String> getClassNames() {
		return classNames;
	}

	public void setClassNames(List<String> classNames) {
		this.classNames = classNames;
	}

	public String getStart_date() {
		return start_date;
	}

	public void setStart_date(String start_date) {
		this.start_date = start_date;
	}

	public String getEnd_date() {
		return end_date;
	}

	public void setEnd_date(String end_date) {
		this.end_date = end_date;
	}

	public String getMonthName() {
		return monthName;
	}

	public void setMonthName(String monthName) {
		this.monthName = monthName;
	}

	public String getClassName() {
		return className;
	}

	public void setClassName(String className) {
		this.className = className;
	}

	public String getPaymentStatus() {
		return paymentStatus;
	}

	public void setPaymentStatus(String paymentStatus) {
		this.paymentStatus = paymentStatus;
	}

	public int getStudentId() {
		return studentId;
	}

	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}

	public String getPaymentMethod() {
		return paymentMethod;
	}

	public void setPaymentMethod(String paymentMethod) {
		this.paymentMethod = paymentMethod;
	}

	public String getPaymentDate() {
		return paymentDate;
	}

	public void setPaymentDate(String paymentDate) {
		this.paymentDate = paymentDate;
	}
	
	
	
}
